### PR TITLE
Fixing test runner

### PR DESCRIPTION
**Pull request by emosesPatreon**
**Fixing test runner**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -2,6 +2,6 @@ require "bundler/gem_tasks"
 require "rake/testtask"
 
 Rake::TestTask.new do |t|
-    t.test_files = FileList["spec/spec_helper", "spec/**/*_test.rb"]
+    t.test_files = FileList["spec/spec_helper", "spec/**/*.rb"]
```
Comment (emosesPatreon) [inline Rakefile]: None of our test files were named *_test.rb.

File: Rakefile [hunk 2]
```diff
@@ -2,6 +2,6 @@ require "bundler/gem_tasks"
 require "rake/testtask"
 
 Rake::TestTask.new do |t|
-    t.test_files = FileList["spec/spec_helper", "spec/**/*_test.rb"]
```
Comment (joshbuddy) [inline Rakefile]: Shouldn't this be `_spec`?

File: spec/api_spec.rb [hunk 3]
```diff
@@ -66,6 +66,10 @@
   end
 
   describe "Patreon::API#fetch_page_of_pledges" do
+    before(:all) do
```
Comment (emosesPatreon) [inline spec/api_spec.rb]: This test was failing

File: Rakefile (hunk 2 above)
Comment (emosesPatreon) [reply Rakefile]: Yeah, I think that'd work.

**Comment by emosesPatreon** [comment]
@joshbuddy Circle is running the tests now, but it's not finding the output at `test_results`.  I've never played with Ruby testing before, do you know how to set the output for `rake test`?

**Comment by emosesPatreon** [comment]
OK, this is working now.

**Comment by joshbuddy** [comment]
@emosesPatreon Nice!

**Pull request merged**